### ISSUE TITLE
Some items in the "all recipes" list overlap

### ISSUE THREAD

**Issue by distress3** [LukeSmithxyz/based.cooking#758]
Tested on mobile, browsers Brave and Chrome. 
See screenshot:
![SmartSelect_20220419-075454_Chrome](https://user-images.githubusercontent.com/36370103/163935831-5e3389e8-1b56-492c-a6c1-8e02d59067cc.jpg)

**Comment by hallzy** [comment]
Seems to be related to the `@media(pointer: coarse)` css.

```css
@media (pointer: coarse) {
	li > a {
	    display: block;
		height: 1.5rem
	}
}
```

I assume that the purpose of that was to give more room on touch screens so it was harder to mis-click a link.

If this should stay in the code, then the height doesn't seem to be big enough for this. Perhaps changing `height` to `min-height` would work, or just remove the height and using a `padding-top` and `padding-bottom` to give the anchor tags a bit more room.

If this shouldn't stay, then removing that css also just fixes the problem as well.

**Comment by lacamera** [comment]
Fixed in #762
